libraries/AP_Scripting/examples/notch_switch.lua

```diff
@@ -0,0 +1,46 @@
+--[[
+
+   example script to switch between two notch setups by changing the
+   attenuation to zero on the notch to disable. This allows for easy
+   in-flight switching between two different notch setups
+--]]
+local INS_HNTCH_ATT = Parameter('INS_HNTCH_ATT')
+local INS_HNTC2_ATT = Parameter('INS_HNTC2_ATT')
+
+if not INS_HNTCH_ATT:get() or not INS_HNTC2_ATT:get() then
+   gcs:send_text(0, string.format("Need 2 notches configured"))
+   return
+end
+
+local last_sw = -1
+local AUX_FN = 300
+
+local attenuation = INS_HNTCH_ATT:get()
+if not attenuation then
+   gcs:send_text(0, string.format("Unable to get attenuation"))
+   return
+end
+
+function update()
+   local sw_current = rc:get_aux_cached(AUX_FN)
```

Comment:
I think this works in lua:
```suggestion
   local sw_current = rc:get_aux_cached(AUX_FN) or 0
```
Although I may be wrong....